modules/ipi-install-checking-ntp-sync.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+// ipi-install-installation-workflow.adoc
+
+:_content-type: PROCEDURE
+[id="checking-ntp-sync_{context}"]
+= Optional: Checking NTP server synchronization
```

Comment:
I know this is done in this assembly already (and some other places), but I think having "Optional:" in front is unnecessary and makes it look a bit odd. Honestly, a lot of the modules in our assemblies are optional if you think about it.

Not something I'd hold up the merge over, but something to reconsider going forward